Software/model/model.py

```diff
@@ -0,0 +1,19 @@
+from Software.model import place
```

Comment:
That should probably be `import place` or better `from place import Place` (needs a minor change in the last line)